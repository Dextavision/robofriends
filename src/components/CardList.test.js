import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('ecpect to render CardList Component', () => {
  const mockrobots = [
    {
      id: 1,
      name: 'house',
      username: 'drhouse',
      email: 'notLupus@gmail.com'
    }
  ]
  expect(shallow(<CardList robots={mockrobots} />)).toMatchSnapshot();
})
