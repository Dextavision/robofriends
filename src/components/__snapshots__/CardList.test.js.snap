// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ecpect to render CardList Component 1`] = `
<div>
  <Card
    email="notLupus@gmail.com"
    id={1}
    key="0"
    name="house"
  />
</div>
`;
